build-system/babel-config/import-resolver.js

```diff
@@ -41,14 +41,33 @@ function readJsconfigPaths() {
   return tsConfigPaths;
 }
 
+/**
+ * Remap external modules that rely on React if building for Preact.
+ * @param {'preact' | 'react'} buildFor
+ * @return {Object}
+ */
+function moduleAliases(buildFor) {
+  if (buildFor == 'preact') {
```

Comment:
Would this behave any different if we always applied it vs. only applying for preact builds?

Reply:
Good thinking. I'll make this is the default